modules/xmpp/JingleSessionPC.js

```diff
@@ -510,8 +511,44 @@ export default class JingleSessionPC extends JingleSession {
                 break;
             }
         };
+
+        /**
+         * The negotiationneeded event is fired whenever we shake the media on the
+         * RTCPeerConnection object.
+         */
         this.peerconnection.onnegotiationneeded = () => {
+            const state = this.peerconnection.signalingState;
+            const remoteDescription = this.peerconnection.remoteDescription;
+
             this.room.eventEmitter.emit(XMPPEvents.PEERCONNECTION_READY, this);
+            if (browser.usesUnifiedPlan() && state.toString() === 'stable'
+                && remoteDescription && typeof remoteDescription.sdp === 'string') {
```

Comment:
style: this needs an extra indent

Reply:
done

Reply:
eslint is complaining about the extra indent so I had to remove it.